packages/auth/src/core/errors.ts

```diff
@@ -359,7 +359,7 @@ export interface ErrorMapRetriever extends AuthErrorMap {
   (): ErrorMap<AuthErrorCode>;
 }
 
-function _prodErrorMap(): ErrorMap<AuthErrorCode> {
+export function _prodErrorMap(): ErrorMap<AuthErrorCode> {
```

Comment:
Nit: I think this gets exported as `prodErrorMap` in the same file?

Reply:
Good catch, thanks!